silkworm/silkrpc/commands/ots_api.hpp

```diff
@@ -59,6 +59,7 @@ class OtsRpcApi {
     boost::asio::awaitable<void> handle_ots_getBlockTransactions(const nlohmann::json& request, nlohmann::json& reply);
     boost::asio::awaitable<void> handle_ots_getTransactionBySenderAndNonce(const nlohmann::json& request, nlohmann::json& reply);
     boost::asio::awaitable<void> handle_ots_getContractCreator(const nlohmann::json& request, nlohmann::json& reply);
+    boost::asio::awaitable<void> handle_ots_traceTransaction(const nlohmann::json& request, nlohmann::json& reply);
```

Comment:
We follow Google's C++ Style Guide with some differences, one of them is `snake_case()` for function names. Please convert these function names, you can do it in another PR if you prefer.